packages/cyberstorm/src/components/Dialog/DialogClose.tsx

```diff
@@ -0,0 +1,18 @@
+"use client";
+import { ReactNode } from "react";
+import * as RadixDialog from "@radix-ui/react-dialog";
+
+export interface DialogCloseProps extends RadixDialog.DialogCloseProps {
```

Comment:
No need to export this?

Also the PropsWithChildren could be used here as well.